### PR TITLE
Fix forecast keys (quantiles) output by TemporalFusionTransformer

### LINKED ISSUE #1944: Incorrect order of quantiles in Temporal Fusion Transformer
## Description
The values in the QuantileForecasts generated by the temporal fusion transformer do not coincide with the forecast_keys. I've looked into the issue and found that the QuantileOutput used in the network sorts the quantiles,

https://github.com/awslabs/gluon-ts/blob/cf58988b53e545962b35631cc0fa943498461fbf/src/gluonts/mx/block/quantile_output.py#L277

however the estimator does not sort the quantiles when creating the QuantileForecastGenerator

https://github.com/awslabs/gluon-ts/blob/cf58988b53e545962b35631cc0fa943498461fbf/src/gluonts/model/tft/_estimator.py#L446-L448

One potential solution could be to sort the quantiles in the QuantileForecastGenerator 
```python
forecast_generator=QuantileForecastGenerator(
                quantiles=sorted([str(q) for q in prediction_network.quantiles])` 
```
The above solution worked for me but I'm not sure if the final fix should be made there or elsewhere.
Note: The estimator internally generates quantiles from num_outputs, and they are disorderd. .e.g, `[0.5, 0.1, 0.9]` for `num_outputs=3`, or `[0.5, 0.1, 0.9, 0.2, 0.8]` for `num_outputs=5`.

## To Reproduce
I took this example from the readme and adapted it simply to show that the order of the forecast_keys is incorrect.

```python
import pandas as pd
from gluonts.dataset.common import ListDataset
from gluonts.model.tft import TemporalFusionTransformerEstimator
from gluonts.mx.trainer import Trainer


url = "https://raw.githubusercontent.com/numenta/NAB/master/data/realTweets/Twitter_volume_AMZN.csv"
df = pd.read_csv(url, header=0, index_col=0)

training_data = ListDataset(
    [{"start": df.index[0], "target": df.value[:"2015-04-05 00:00:00"]}],
    freq = "5min"
)

estimator = TemporalFusionTransformerEstimator(freq="5min", 
                                               prediction_length=2, 
                                               trainer=Trainer(epochs=10))
predictor = estimator.train(training_data=training_data)

test_data = ListDataset(
    [{"start": df.index[0], "target": df.value[:"2015-04-15 00:00:00"]}],
    freq = "5min"
)

forecast_it = predictor.predict(test_data)
print(next(iter(forecast_it)))
print(predictor.prediction_net.output.quantiles)
```

## Error message or code output
```
QuantileForecast(array([[43.30777 , 43.03252 ],
       [48.941864, 48.626392],
       [56.510143, 56.666008]], dtype=float32)), start_date=Timestamp('2015-04-15 00:00:00', freq='5T'), freq='5min', forecast_keys=['0.5', '0.1', '0.9'], item_id=None, info=None)
[0.1, 0.5, 0.9]
```

## Environment
- Operating system: Ubuntu 18.04.6 LTS
- Python version: 3.9.6
- GluonTS version: 0.9.3
- MXNet version: 1.9.0

### PR DESCRIPTION
**Pull request by lostella**
*Issue #, if available:* Fixes #1944

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Comment by jaheba** [comment]
Can we add a test?

**Review by jaheba — APPROVED** [review]
